models/contacts.js

```diff
@@ -35,9 +36,9 @@ const removeContact = async (contactId) => {
 
 const addContact = async (body) => {
     try {
-        await contactSchema.validateAsync(body);
+        await contactSchema.validateAsync(body, body.owner);
         body.email = body.email.toLowerCase();
-        return await Contact.create(body);
+        return await Contact.create(body, body.owner);
```

Comment:
Kiedy uzywasz Contact.create(contact); To ten contact object powinien juz mieć klucz owner. 
```
const contact = {
  ...inne klucze, 
  owner
  };
  ```